react/features/base/profile/middleware.js

```diff
@@ -31,13 +34,11 @@ MiddlewareRegistry.register(store => next => action => {
  * @returns {void}
  */
 function _updateLocalParticipant(store) {
+    const localParticipant = getLocalParticipant(toState(store));
     const profile = getProfile(toState(store));
 
-    const newLocalParticipant = {
-        email: profile.email,
-        local: true,
-        name: profile.displayName
-    };
+    localParticipant.email = profile.email;
+    localParticipant.name = profile.displayName;
```

Comment:
Ouch! Did we just modify the redux state here?

Reply:
Got the feedback to do it like this - assumed that I was asked to do this since this is only a copy of the store (or was it your point back then @saghul ? not entirely sure because we didn't discuss it). Originally the object was deconstructed and a new instance created (see above).

Reply:
Probably misread, my bad! Thanks for catching that @lyubomir !